src/pages/indicatorTypes/form/IndicatorTypesSchema.ts

```diff
@@ -4,7 +4,14 @@ import { getDefaults, createFormValidate, modelFormSchemas } from '../../../lib'
 const { identifiable } = modelFormSchemas
 
 export const IndicatorSchema = identifiable.extend({
-    factor: z.coerce.number().int(),
+    factor: z.coerce
+        .number({ invalid_type_error: 'Please enter a number' })
+        .int()
+        .min(1, 'Please enter a value above 0')
```

Comment:
I think that in the [schema](https://debug.dhis2.org/dev/api/schemas/indicatorType) the min is -2147483648, so we prob should allow negative numbers. 
we can maybe use this?

```
z.number().int().refine((value) => value !== 0, {
  message: "Number must not be zero",
});
```

Reply:
Thanks for catching that, this should be resolved now.